src/components/nodes/subProcess/index.js

```diff
@@ -13,10 +14,10 @@ export default {
   control: false,
   category: 'BPMN',
   icon: require('@/assets/toolpanel/subProcess.svg'),
-  label: 'Sub Process',
+  label: defaultNames[id],
```

Comment:
Just wondering - why is the label untranslated but the moddle name translated?

Reply:
`$t` isn't available here. We would need another solution to translate them.